qt_event_handler.py

```diff
@@ -0,0 +1,50 @@
+""" QtEventHandler handles the events from QWidgets """
+
+# !/usr/bin/python3
+from itasks_components import ItasksComponent
+
+
+class QtEventHandler:
+    """ QtEventHandler handles the events from QWidgets """
+
+    def __init__(self, itasks_service):
+        """
+        :param itasks_service: iTasks service
+        """
+        self.itasks_service = itasks_service
+
+    def button_clicked_event(self, itasks_component: ItasksComponent):
+        """ button clicked event handler """
```

Comment:
python docstring params are missing (for consistency with other classes)

Reply:
added params to docstrings